app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -121,6 +127,10 @@ fun MapScreen(
                   // recenter the camera on the marker if it is not the case already
                   mapViewportState.setCameraOptions { center(it.point) }
 
+                  // get the data from the PointAnnotation and deserialize it
+                  val parkingData = it.getData()?.asJsonObject
+                  val parking_deserialized = gson.fromJson(parkingData, Parking::class.java)
```

Comment:
Small comment... We try to use camelCase :sweat_smile:

Reply:
Oups mistake on my part . fixed !